src/common/options.cc

```diff
@@ -3582,8 +3582,9 @@ std::vector<Option> get_global_options() {
 
     Option("osd_objectstore", Option::TYPE_STR, Option::LEVEL_ADVANCED)
     .set_default("filestore")
+    .set_enum_allowed({"bluestore", "filestore", "memstore", "kstore"})
     .set_flag(Option::FLAG_CREATE)
-    .set_description(""),
+    .set_description("back end type for an OSD (like filestore or bluestore)"),
```

Comment:
s/back end/backend/ here ("backend" is a standard IT technical term - if in doubt, grep in `doc/`)